ui/components/app/snaps/snap-ui-footer-button/snap-ui-footer-button.tsx

```diff
@@ -26,6 +26,7 @@ import { getHideSnapBranding } from '../../../../selectors';
 type SnapUIFooterButtonProps = {
   name?: string;
   variant?: ButtonVariant;
+  snapVariant?: ButtonVariant & 'destructive';
```

Comment:
Do we not have an exported type for the button variants in the SDK?

Reply:
Can we use `ButtonProps['variant']` then?

Reply:
> I think no. ButtonVariant still has no `destructive` variant. It looks like this:
> 
> ```ts
> export enum ButtonVariant {
>   Primary = 'primary',
>   Secondary = 'secondary',
>   Link = 'link',
> }
> ```

This is the wrong one, this one is from extension, `ButtonProps` should be coming from `snaps-sdk`

Reply:
Yeah, agreed ^